### PR TITLE
nixos/pantheon: add docs

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
I finally translated the docs I wrote for Pantheon in markdown months ago :confetti_ball: 
https://gist.github.com/worldofpeace/27fcdcb111ddf58ba1227bf63501a5fe

There's one thing I couldn't translate and it's the cool citation/reference `[^1]`, which in markdown would give you a nice hover text and citation at the end of the section (clickable link).
###### Things done

Manual was built.

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/modules/services/x11/desktop-managers/pantheon.xml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+     </para>
+    </listitem>
+   </varlistentry>
+
+   <varlistentry xml:id="sec-pantheon-faq-appcenter">
+    <term>Does Appcenter work, or is it available?</term>
+    <listitem>
+     <para>
+      Appcenter has been available since 20.03, but it is of little use. This is because there's no functioning Packagekit
+      backend for Nix 2.0. In the near future you will be able to install Flatpak applications from Appcenter on NixOS.
```
Comment (jtojnar) [inline nixos/modules/services/x11/desktop-managers/pantheon.xml]:
```suggestion
      backend for Nix 2.0. In the near future you will be able to install Flatpak applications from AppCenter on NixOS.
```

File: nixos/modules/services/x11/desktop-managers/pantheon.xml [hunk 2]
```diff
@@ -0,0 +1,158 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xml:id="sec-pantheon">
```
[the remark this answers is not in the document]
Comment (worldofpeace) [reply nixos/modules/services/x11/desktop-managers/pantheon.xml]: I should only substitute this id?
(I hope it's apparent, my knowledge of docbook is purely imitation :rofl: )

[the remark this answers is not in the document]
Comment (jtojnar) [reply nixos/modules/services/x11/desktop-managers/pantheon.xml]: It is just a convention, DocBook does not care. See `gnome.nix` for an example how I handled the `id`s in the nested sections.

**Comment by worldofpeace** [comment]
@jtojnar I wasn't sure how to link into the nixpkgs manual for the overrides.

**Comment by worldofpeace** [comment]
I also wasn't sure about plugins vs plug-ins. I see a lot of docs with `plugins` and I almost always use this spelling.

**Pull request merged**